source/guide_gitea.rst

```diff
@@ -96,93 +157,276 @@ and a new config file at ``~/gitea/custom/conf/app.ini`` to specify the port and
   [service]
   DISABLE_REGISTRATION = true
 
-Gitea provides many other configuration options. Take a look at the Documentation_
-to learn more.
+When using this, we have to finish the installation via gitea web service https://isabell.uber.space/install. This is exposed without any password request. We improve the configiguration with some modifications, e.g.:
+
+* Filling the database access data that we would otherwise enter in the web installation step. (``[database]`` section)
+* As security feature we lock the web installation and change the default password complexity to allow well to remember and secure passwords. (See `XKCD No. 936 <https://xkcd.com/936/>`_  and `Explained XKCD No. 936 <https://explainxkcd.com/wiki/index.php/936:_Password_Strength>`_ for the math behind it. 😉 ``[security]`` section)
+* We disallow public registration and set some privacy settings. (``[service]`` section)
+* Naming the repositories path.
+* Set the eye-friendly dark theme as default theme, set mailing, set time and set logging options.
+
+For more informations about the possibilities and configuration options see the Gitea documentation_ and the Gitea `configuration sample <https://github.com/go-gitea/gitea/blob/master/custom/conf/app.ini.sample>`_.
 
-Configure web server
---------------------
+.. note:: The minimum improvements are written in the sections of ``[database]``, ``[security]`` and ``[service]``.
 
-.. note::
+.. warning:: Replace ``isabell`` with your username, fill the database password ``PASSWD =`` with yours and enter the generated random into ``SECRET_KEY =``.
 
-    Gitea is running on port 9000.
+.. code-block:: ini
 
-.. include:: includes/web-backend.rst
+  APP_NAME = Gitea
```

Comment:
Like with the previous PR, this is the full configuration including all default values (e.g. `OFFLINE_MODE`, `PASSWORD_HASH_ALGO` and many more). Please remove them and only include configuration that actually needs changing. Having the whole default config in where makes it hard to update the guide in the future.

Reply:
Hopefully it's slim enough now. :-) 

I think some security improvements (e.g. to allow lower case passwords only as referred to xkcd) are ok. Email confirmation depends to configure [mailer] section.

Reply:
yes, any security improvements are certainly welcome and should stay! The lines that just set the default values are the ones that bothered me :sweat_smile:

Reply:
I checked again. 

**`[server]`**
I removed the `LFS` options complete. The default paths are looking good, and the security key will be created in on of the next steps, so we can remove this here.

I think `OFFLINE_MODE = true` is an important privacy option, so there will be no traffic to gravatar and others.

**`[service]`** and **`[mailer]`**
I think the options written there a a combination of security, so the platform is not open for any others by default and keeping privacy of new users by default.